cylc/uiserver/__init__.py

```diff
@@ -13,7 +13,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-__version__ = "1.1.0"
+__version__ = "1.1.1.dev"
```

Comment:
1.2.0.dev?

Reply:
That isn't showing up when you choose all commits. It's on 1.2.0.dev